javascript-and-json/main.js

```diff
@@ -0,0 +1,22 @@
+var books = [
+  {
+    title: 'Lord of the rings',
+    author: 'JRR Tolkien',
+    isbn: '123456789101'
+  },
+  {
+    title: 'The Hobbit',
+    author: 'JRR Tolkien',
+    isbn: '223355779900'
+  },
+  {
+    title: 'The Silmarillion',
+    author: 'JRR Tolkien',
+    isbn: '109876543211'
+  }];
+console.log(JSON.stringify(books));
+
+console.log(JSON.parse(JSON.stringify({
+  sudent: 'Cooper',
+  number: 'id'
```

Comment:
This is not the intended behavior, you don't want to need to run JavaScript methods to create your JSON string for this step, it should be written directly into the file as a string stored into a variable.